src/sentry/notifications/platform/discord/provider.py

```diff
@@ -96,4 +96,14 @@ def is_available(cls, *, organization: RpcOrganizationSummary | None = None) ->
 
     @classmethod
     def send(cls, *, target: NotificationTarget, renderable: DiscordRenderable) -> None:
-        pass
+        from sentry.integrations.discord.integration import DiscordIntegration
+
+        if not isinstance(target, cls.target_class):
```

Comment:
There should be a way we can guarantee this instance in some shared code vs having to perform instance checks everywhere. Let's workshop this on Monday.